### PR TITLE
[MTDSA-11772] - dependency upgrades

### PR DESCRIPTION
**Pull request by DreamcastKnight**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -22,17 +22,17 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % "5.20.0",
+    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % "5.24.0",
     "uk.gov.hmrc"                  %% "simple-reactivemongo"      % "8.0.0-play-28",
```
Comment (BetaDraconis) [inline project/AppDependencies.scala:26]: Can you please migrate this to hmrc-mongo https://github.com/hmrc/hmrc-mongo/blob/main/MIGRATION.md

**Review by BetaDraconis — CHANGES_REQUESTED** [review]
sbt-plugin -> 2.8.15
simple-reactive-mongo -> hmrc-mongo

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -22,17 +22,18 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % "5.20.0",
-    "uk.gov.hmrc"                  %% "simple-reactivemongo"      % "8.0.0-play-28",
-    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.13.1"
+    "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % "5.24.0",
+    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.13.2",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"      % "0.63.0",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-test-play-28" % "0.63.0"
```
Comment (james-work-account) [inline project/AppDependencies.scala:28]: Move to the test dependencies

**Review by james-work-account — CHANGES_REQUESTED** [review]

File: project/AppDependencies.scala (hunk 2 above)
Comment (DreamcastKnight) [reply project/AppDependencies.scala:28]: where is that, sorry?

**Pull request merged**